### DISCUSSION TITLE
changes to navigation keymaps behavior?

### DISCUSSION THREAD

**Discussion by redxtech** [LazyVim/LazyVim#814] [General]
I have been using `<C-w>` as a keymap for `daw`, via `map('n', '<C-w>', 'daw', { desc = 'Delete word' })` (map is the same function as in `/lua/lazyvim/config/keymaps.lua`), and it broke when I updated today. Instead of navigating windows like it used to, it would apply `daw`.

I looked into the commits, and it looks like this is the commit that broke that behavior by adding `remap = true` to the `map` call: https://github.com/LazyVim/LazyVim/commit/5bf45e3139aa3a8cf8e1bf9fd2a2d422fe1c6cf7.

It looks like this is now the intended behavior, so is my only option here to re-implement these navigation keybinds in my own `keymaps.lua` with the `remap = false`?

**Comment by folke** [comment]
Good point. I reverted those changes

**Comment by redxtech** [comment]
Perfect, thank you!
